chsarp/DefaultLambdaParameters.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Microsoft.Extensions.Logging;
+
+namespace CSharp;
+
+public class DefaultLambdaParameters(ILogger<CollectionExpressions> _logger)
+{
+    public void DefaultParameters()
+    {
+        var incrementValue = (int value = 2) =>
+        {
+            _logger.LogInformation($"{(value == 2 ? "Called with default value" : "Called with given value")}: {value}");
+
+            return value + 1;
+        };
+
+        incrementValue();
+        incrementValue(5);
+
+        var action = incrementValue;
+        action();
+    }
+
+    public void ParamsArrayParameters()
```

Comment:
Is this a method related to the class it is in? It feels like the class name is different with what it does.